src/main/java/edu/kit/ipd/crowdcontrol/objectservice/payment/PaymentDispatcher.java

```diff
@@ -56,8 +64,18 @@ private void dispatchPayment(Experiment exp) {
         paymentCalc.estimatePayment(exp).entrySet().stream()
                 .collect(Collectors.groupingBy(
                         entry -> entry.getKey().getPlatform(),
-                        Collectors.mapping(entry -> new PaymentJob(entry.getKey(), entry.getValue(), ""), Collectors.toList())
-                ))
+                        Collectors.mapping(new Function<Map.Entry<WorkerRecord,Integer>, PaymentJob>() {
+                            @Override
+                            public PaymentJob apply(Map.Entry<WorkerRecord, Integer> workerRecordIntegerEntry) {
+                                String message = "";
+                                try {
+                                    message = feedbackCreator.getFeedback(exp.getId(), workerRecordIntegerEntry.getKey().getIdWorker());
+                                } catch (FeedbackException e) {
+                                    log.fatal("Creating feedback message of experiment " + exp.getTitle() + " could not take place!", e.getCause());
```

Comment:
I am talking about this line here, why not just pass e if you cannot assert that there is always a cause? So you dont loose information what exactly caused the problem.